lib/decoders/kmz.js

```diff
@@ -49,6 +48,32 @@ class KMZ extends Duplex {
   }
 
   _onBuffered() {
+    this.emit('readable');
+  }
+
+  _onOpenKmlStream(kmlStream, zipFile) {
+    kmlStream
+      .pipe(this._kmlDecoder)
+      .on('error', (err) => {
+        this.emit('error', err);
+      })
+      .on('data', (data) => {
```

Comment:
`push` has a side effect of pushing the datum onto the stream, so it's not a noop. It's just that if `push(data) == true` then that means the stream's high water mark hasn't been hit, so we can keep pushing stuff. If push is `false` then that means the stream is being overwhelmed and we need to pause it and resume it when a `drain` event is fired.